lib/diagnostic.rb

```diff
@@ -5,36 +5,40 @@ class Diagnostic
   # enumerable supplied.  Assume each entry implements a length
   # method that returns a Fixnum.
   def longest(enum)
-    # Your code goes here
+    enum.max
   end
+  # Jason Liang's Comment:
+  # I wanted to write enum.max.length but each entry
+  # implements a length method.
+
```

Comment:
The way to accomplish this is to either 
(a) use `max_by` and pass it a block -- e.g. `enum.max_by { |entry| entry.length }` -- or
(b) use `max` with a custom comparator block built on top of the spaceship operator,
e.g. `enum.max { |a, b| a.length <=> b.length }`